src/panel-manager.js

```diff
@@ -86,6 +94,7 @@ genotet.panelManager.activatePanel_ = function(viewID) {
  * Creates a panel with the given name.
  * @param {!genotet.View} view Name of the view.
  * @return {!jQuery}
+ * @this
```

Comment:
@this usually points to a class instance. genotet.panelManager is a namespace, not a class instance.